### PR TITLE
Update summarizeoperator.md

### PR DESCRIPTION
**Pull request by ziham1531991**

### CODE REVIEW

**Comment by spektom** [comment]
Do we want to add count_distinct() to the list?

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/summarizeoperator.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The following table summarizes the default values of aggregations:
 
 | Operator | Default value |
 |--|--|
-| `count()`, `countif()`, `dcount()`, `dcountif()` | 0 |
+| `count()`, `countif()`, `dcount()`, `dcountif()`, `sum`, `sumif`, `variance`, `varianceif`, `stdev`, `stdevif` | 0 |
 | `make_bag()`, `make_bag_if()`, `make_list()`, `make_list_if()`, `make_set()`, `make_set_if()` | empty dynamic array              ([]) |
 | All others | null |
```
Comment (zivcaspi) [inline data-explorer/kusto/query/summarizeoperator.md:70]: We also need to add this comment to each aggregate function individually, and to the "null handling" topic page

**Comment by v-dkorn** [comment]
You can close this PR, as I'm incorporating these changes along with more in the following new PR:

https://github.com/MicrosoftDocs/dataexplorer-docs-pr/pull/5296

Thanks so much!

**Comment by YaelSchuster** [comment]
#please-close

**Pull request closed without merging**